src/dotnet/commands/dotnet-migrate/LocalizableStrings.cs

```diff
@@ -30,5 +30,7 @@ internal class LocalizableStrings
         public const string CmdSkipBackupDescription = "Skip moving project.json, global.json, and *.xproj to a `backup` directory after successful migration.";
 
         public const string MigrationFailedError = "Migration failed.";
+
+        public const string MigrationAdditionalHelp = "Your project has been migrated to .csproj format and can be used with this build of .NET Core Tools.  Please visit https://aka.ms/coremigration to report issues or for help.";
```

Comment:
Reads to me like there's a few words missing.

"Your project has been migrated to **the** .csproj format and can be used with this build of **the** .NET Core Tools.  Please visit https://aka.ms/coremigration to report issues or **ask** for help."